packages/react/src/lib/felaExpandCssShorthandsPlugin.ts

```diff
@@ -0,0 +1,33 @@
+import * as _ from 'lodash'
+import * as _expand from 'css-shorthand-expand'
+
+// `css-shorthand-expand` is a CJS library, there are known issues with them:
+// https://github.com/rollup/rollup/issues/1267#issuecomment-446681320
+const expand = (_expand as any).default || _expand
+
+export default () => {
+  const expandCssShorthands = styles => {
```

Comment:
NIT: type for `styles`

Reply:
I checked fela's plugins, and they use here Object as a type.. Not best, but will add it..